docs/V1/Customer/get_respondents.md

```diff
@@ -48,37 +54,94 @@ POST https://api.scoringline.com/api-saas/v1/customer/questionnaires/foobar/resp
 
 Avec les paramètres, ci-dessous.
 
-Clé                        | valeur                              | description
----------------------------|-------------------------------------|---------------------------------
-filters[status]            | accepted                            | Par statut des répondants
-filters[customStatus]      | (integer-string-integer[]-string[]) | Passer les identifiants des statuts personnalisés
-filters[createdfrom]       | (integer) 1451602800                | Nouvelles candidatures depuis 
-filters[lastUpdatedFrom]   | (integer) 1466066108                | Candidatures mises à jour depuis
+Clé                        | Valeur       
+---------------------------|--------------
+filters[status]            | `accepted`   
+filters[createdfrom]       | `1451602800` 
```

Comment:
I had to search for this value, you should let the `integer` thing and maybe add `(UNIX timestamp)` as seen above.